motor/core.py

```diff
@@ -835,6 +860,18 @@ async def f():
               async for doc in collection.aggregate(pipeline):
                   print(doc)
 
+        Note that this method returns a :class:`MotorCommandCursor` which
+        lazily runs the aggregate command when first iterated. In order to run
+        an aggregation with ``$out`` or ``$merge`` the application needs to
+        iterate the cursor, for example::
+
+           cursor = motor_coll.aggregate([{'$out': 'output-collection'}])
```

Comment:
I think we should also show `async for ...: pass` ?

Reply:
Done.